modules/test/playwright/tools/test-setup.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+function copy_portal_ext_properties() {
+	for file in \
+		"${PROJECT_DIR}/modules/test/playwright/tests/${TEST_PROJECT}/env/portal-ext.properties" \
+		"${PROJECT_DIR}/modules/test/playwright/env/portal-ext.properties"
+	do
+		if [ -f "${file}" ]
+		then
+			echo "  ===> ${file}"
+
+			cp "${file}" "${BUNDLES_DIR}"
```

Comment:
would'nt this overwrite one file with the other? same name, same target directory. Is this intended?

Reply:
Outdated: I have removed the bash scripts.